docs/wks-and-footloose.md

```diff
@@ -18,19 +18,21 @@
    export BACKEND=docker
    ```
 
-1. Install [`footloose`](https://github.com/weaveworks/footloose):
+1. Install [footloose](https://github.com/weaveworks/footloose):
```

Comment:
was this change a mistake maybe?  It seems appropriate to use code blocks here because it's talking about `footloose` the binary and not footloose the product (codeblocks are used everywhere else this way).

Reply:
Without my change, the user doesn't know this is a link.

Reply:
I'm sorry, I don't follow: the text becomes blue and the cursor changes on hover.  Can you be more specific please?

For example, [in this section in some of the eksctl docs](https://github.com/weaveworks/eksctl/blob/master/site/content/gitops-quickstart/01-apply-gitops.md#your-gitops-cluster) it's done 3 times in a row under the third bullet (Prometheus).